src/components/common/DropDown/DropDownSelector.tsx

```diff
@@ -0,0 +1,96 @@
+import React, {useState, useRef, useMemo} from 'react';
+import {
+  TouchableOpacity,
+  FlatList,
+  ScrollView,
+  Keyboard,
+  TouchableWithoutFeedback,
+} from 'react-native';
+import S from './DropDownSelector.style';
+
+type Props = {
+  value: string[];
+  options: string[];
```

Comment:
```suggestion
type Props = {
  values: string[];
  options: string[];
```

다중 선택이 가능하니 values 는 어떤가요